### ISSUE TITLE
Question: Can I use the default empty world when calling steps from steps?

### ISSUE THREAD

**Issue by joelindridge-sh** [vitalets/playwright-bdd#255]
Hi @vitalets, thanks for this project, it's working really well.

I noticed a default empty world was implemented for pw-style steps: https://github.com/vitalets/playwright-bdd/issues/208#issuecomment-2277492327

How can I make use of this default empty world when calling steps from steps?

(using very latest `"playwright-bdd": "^8.0.0"`.)

---

In this example, Test1 passes, but Test2 fails.

test.feature:
```gherkin
Feature: Test

  Scenario: Test1
    When a
    Then b

  Scenario: Test2
    When c
```
test.ts:
```ts
import { createBdd } from "playwright-bdd";
import { expect } from "@playwright/test";

const { When, Then } = createBdd();

const a = When("a", async function ({}) {
  this.a = 1;
});

const b = Then("b", async function ({}) {
  expect(this.a).toBe(1);
});

When("c", async function ({}) {
  await a({});
  await b({});
});
```

![image](https://github.com/user-attachments/assets/15dbc0dc-bfff-445c-89b7-e8c4024c6c32)

**Comment by vitalets** [comment]
Hi @joelindridge-sh 

This way:
```js
When("c", async function ({}) {
  await a.call(this);
  await b.call(this);
});
```

Will add it to docs.

**Comment by joelindridge-sh** [comment]
Thanks @vitalets, but it seems this doesn't work in my playwright-style example. I guess this will only work with cucumber-style steps.

**Comment by vitalets** [comment]
It should work with playwright-style as well, if you don't use _arrow functions_ for step definitions.

**Comment by vitalets** [comment]
Confirm the bug, releasing as 8.0.1

**Comment by vitalets** [comment]
Released in 8.0.1. 
Now the following sample works:
```ts
const a = When("a", async function () {
  this.a = 1;
});

const b = Then("b", async function () {
  expect(this.a).toBe(1);
});

When("c", async function () {
  await a.call(this);
  await b.call(this);
});
```

**Comment by vitalets** [comment]
> I think I can also define the default world type like this (I'm not sure if there's an easier way):

If it contains only `a` then it can be:
```ts
interface MyWorldType {
  a: number;
}
```
